common/src/main/java/com/microsoft/identity/common/internal/ui/webview/certbasedauth/OnDeviceCertBasedAuthChallengeHandler.java

```diff
@@ -123,15 +122,7 @@ public void alias(String alias) {
      */
     @Override
     public void cleanUp() {
-        final String methodTag = TAG + ":cleanUp";
-        //For on-device CBA, we need to clear the certificate choice cache here so that
-        // the user will be able to login with multiple accounts with CBA
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-            WebView.clearClientCertPreferences(null);
-        } else {
-            Logger.warn(methodTag, "Client Cert Preferences cache not cleared due to SDK version < 21 (LOLLIPOP). " +
-                    "Subsequent CBA attempts will fail due to the cached action, so the user must restart the app before attempting to login with CBA again.");
-        }
+       //nothing needed here
```

Comment:
Is there a way to check if cleanup has already been performed, and if not, we perform the clean up here.

Reply:
Potentially, though I might risk calling clearclientcertpreferences more than I would need to if I'm not careful about the logic. And that would raise more cert_changed issues on the OneAuth side, which is why I've been moving the method call around in the first place.
If you don't mind, I'll look into adding logic for that in a subsequent PR.